src/layouts/Post/Post.js

```diff
@@ -19,6 +19,11 @@ import styles from './Post.module.css';
 export const Post = ({ children, title, date, abstract, banner, timecode, ogImage }) => {
   const scrollToHash = useScrollToHash();
   const imageRef = useRef();
+  const [dateTime, setDateTime] = useState(null);
+  
+  useEffect(() => {
+    setDateTime(formatDate(date));
+  }, [dateTime]);
```

Comment:
This project uses the exhaustive deps hooks eslint rule, so we should include `date` in this dependency array

```suggestion
  }, [date, dateTime]);
```